### DISCUSSION TITLE
How to add MAN page?

### DISCUSSION THREAD

**Discussion by Zabrane** [goreleaser/nfpm#339] [Q&A] [answered]
Hey guys

I would like to ship a `man page` for my application within the  generated the `DEB/RPM`.
Is there a way to achieve that with `nfpm`?

Thanks

**Comment by caarlos0** [comment, marked as answer]
I think you can just package the man files and put them in `manpath` (e.g. `/usr/share/man`).

**Reply by Zabrane** [reply]
> I think you can just package the man files and put them in `manpath` (e.g. `/usr/share/man`).

Amazing. Thanks @caarlos0

**Reply by caarlos0** [reply]
Let me know how it works! If it does we can properly document it :D

**Reply by Zabrane** [reply]
> Let me know how it works! If it does we can properly document it :D

As easy as (a single `man` page here):
```yaml
contents:
- src: app.1.gz
  dst: /usr/share/man/man1/app.1.gz
```

Alternatively, you can use a tarball `app.1.tgz` in case of multiple man pages.

**Reply by caarlos0** [reply]
nice, thanks!
